tool/state-processor/src/system/README.md

```diff
@@ -34,7 +34,8 @@
   - compare the calculated one with the storage one
 
 - some remaining accounts, bridge endpoint accounts - TODO
-- special accounts - TODO
-  - parachain backing account 0x8c585F9791EE5b4B23fe82888cE576DBB69607eB
-  - bridge root account 0x726f6f7400000000000000000000000000000000
-  - pallet accounts
+- special accounts
+  - parachain backing account DONE
```

Comment:
Question: Which is the parachain backing account?

Reply:
#218